databuilder/extractor/minio_extractor.py

```diff
@@ -0,0 +1,92 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+import csv
+import importlib
+# from collections import defaultdict
+
+from pyhocon import ConfigTree
+from typing import Any
+import boto3
+
+from databuilder.extractor.base_extractor import Extractor
+from databuilder.models.table_metadata import TableMetadata, ColumnMetadata
+
+class MinioExtractor(Extractor):
+
+    """
+    An Extractor that extracts records via CSV.
```

Comment:
Class comment should describe more than just CSV -- `via CSV` is also very vague.